api/CanvasRenderingContext2D.json

```diff
@@ -3758,19 +3758,19 @@
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "version_added": true
             },
             "opera_android": {
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": true
             },
             "webview_android": {
               "version_added": false
```

Comment:
This one should also so `true`.

Reply:
"This one" means the entry for "webview_android", I think.

Reply:
Yes.

Reply:
Fixed, thanks for the clarification.